### PR TITLE
Update screenshot in readme

### PR DESCRIPTION
**Pull request by kolibril13**
In the readme, there is currently a picture showing an astronaut in the napari viewer, but the script generates cells in 3d.
Therefore, I've changed the image to a screenshot that I get when I run the example script in the latest napari version.
I've used the GitHub markdown drag and drop to add this image, so this image lives in the GitHub image database, and does not live in this repo. This might be a good idea, as the screenshot does not use repo space that way.
The outdated screenshot got originally readded in https://github.com/napari/napari/pull/5220

### CODE REVIEW

**Review by psobolewskiPhD — APPROVED** [review]

File: README.md
Comment (Carreau) [inline README.md]: There does not seem to be anything else except this that link to githubusercontent, I don't have any particular preference, but I want to make sure other are Ok with that.

Comment (psobolewskiPhD) [reply README.md]: The other option would be to link to the docs repo?
https://github.com/napari/docs/blob/d1c287825d065a3d7511979f97f9194d63d6da7d/docs/images/multichannel_cells.png

Comment (psobolewskiPhD) [reply README.md]: that's the image used in napari.org
https://napari.org

**Review by psobolewskiPhD — CHANGES_REQUESTED** [review]
I was hasty. I prefer doing back to the cells image, but I think we should use the actual image from the napari/docs repo, as @jni requested in the other PR that was merged and went back to astronaut.
So just replace `screenshot-add-image.png` with `multichannel_cells.png`

Also, we should correct, not drop, the alt text. I suggest using that also from the other repo:
```
![napari viewer with a multichannel image of cells displayed as two image layers: nuclei and membrane.]
```

File: README.md
Comment (psobolewskiPhD) [inline README.md]:
```suggestion
![napari viewer with a multichannel image of cells displayed as two image layers: nuclei and membrane.](https://github.com/napari/docs/blob/main/docs/images/multichannel_cells.png)
```
I think this is the preferred way of fixing this.

Comment (kolibril13) [reply README.md]: sounds good.
I just want to note that now the generated image has other parameters than given from the readme example code.
 The actual code that generates this new example is here: 
https://napari.org/stable/howtos/layers/image.html#loading-multichannel-images

Comment (psobolewskiPhD) [reply README.md]: You're right of course. Then again looking more closely, the code block on that page also doesn't produce that image, because it's not making the viewer 3D.
But yeah, the contrast limits are changed to make it a bit contrast-ier and it's rotated to show the 3D.

Comment (psobolewskiPhD) [reply README.md]: Anyhow, for me it's fine. Maybe @jni or @GenevieveBuckley can take a look and make the final call whether we go with that image (which is also on napari.org) or make a new one or use yours (but add it to the napari/docs repo), as they authored the PR that changed to cells3D.

**Comment by psobolewskiPhD** [comment]
Tests must be un-related, I'l re-run

**Comment by GenevieveBuckley** [comment]
I'm +1 @psobolewskiPhD

**Review by psobolewskiPhD — APPROVED** [review]

**Review by jni — APPROVED** [review]

File: README.md
Comment (kolibril13) [reply README.md]: > contrast limits are changed to make it a bit contrast-ier and it's rotated to show the 3D.  

I think it's great that it looks shiny with these tweaked parameters, and at the same it's good that the example code is kept at the bare minimum. This PR can be merged from my side as well :)

**Pull request merged**
